### ISSUE TITLE
Multiplayer

### ISSUE THREAD

**Issue by usach3v** [studio-minus/ppg-bugs-and-requests#784]
**Feature description**
Everyone has their own preference, but maybe the creator of the server will be able to assign the camera of the 2nd player to himself. The field of vision was created more for games of the type: guess which cell the object is in, or the person will be torn to pieces. Mods are installed by the Host, before starting the server, there will be something in the menu (see menu picture1). In this menu, the player(s) download Host mods, relogin, wait for compiling, and are already logged into the server. The server will shutdown immediately after the host disconnects/leaves the game. (If you wish, you can add shared servers (100 slot), which will have 1 admin, or there will be some kind of mode.

**Feature purpose**
If you count private servers, for playing with friends, it's more pleasant to create something voluminous together. Increase the number of works in the workshop. But the main purpose of the "private" servers is rather to have fun with friends, which will give the game the "Co-op" label. Shared or "dedicated" servers can serve as a developer-created mode that gives the game the "Vs Players" tag.

**Additional context**
I tried to make a concept for the multiplayer menu. do not judge strictly.
![picture1](https://user-images.githubusercontent.com/101024315/156903099-a4c9cf12-12b6-4617-aa3e-ad6f6392335e.png)

**Comment by UniDuki** [comment]
Ignoring the fact that the whole game would probably have to be rewritten for this to even be remotely possible.
This is another duplicate of #194. That's 4 times that multiplayer has been suggested here now.

**Comment by D3stStrut** [comment]
> Ignoring the fact that the whole game would probably have to be rewritten for this to even be remotely possible. This is another duplicate of #194. That's 4 times that multiplayer has been suggested here now.

SHUT UP, SHUT UP!
